From ea53a7f42c1c5da688ef79e53731dcc3810a1256 Mon Sep 17 00:00:00 2001
From: Namhyung Kim <namhyung@gmail.com>
Date: Sun, 6 Jul 2025 15:13:30 -0700
Subject: [PATCH] build: Fix a build error on i386

It happened to have common.S as well as common.c, so it picked the .S
before .c for the uftrace-arch.a.  And it caused undefined symbols.

  /usr/bin/ld: libmcount/symbol.op: in function `load_dynsymtab':
  utils/symbol.c:722:(.text+0x2b19): undefined reference to `uftrace_arch_ops'
  libmcount/symbol.op: in function `load_elf_dynsymtab':
  utils/symbol.c:586:(.text+0x184a): undefined reference to `uftrace_arch_ops'

Fixed: #2010

Upstream-Status: Backport [https://github.com/namhyung/uftrace/commit/ea53a7f42c1c5da688ef79e53731dcc3810a1256]
Signed-off-by: Namhyung Kim <namhyung@gmail.com>
---
 arch/i386/{common.S => thunk.S} | 0
 1 file changed, 0 insertions(+), 0 deletions(-)
 rename arch/i386/{common.S => thunk.S} (100%)

diff --git a/arch/i386/common.S b/arch/i386/thunk.S
similarity index 100%
rename from arch/i386/common.S
rename to arch/i386/thunk.S
